cuda_core/tests/test_event.py

```diff
@@ -80,3 +80,63 @@ def test_is_done(init_cuda):
     # Without a sync, the captured work might not have yet completed
     # Therefore this check should never raise an exception
     assert event.is_done in (True, False)
+
+
+def test_error_timing_disabled():
```

Comment:
I'd insert the new tests right after the existing `test_timing`.

Looking at the existing test with a fresh eye, I believe this needs to be unindented (i.e. the existing test is incorrect):

* https://github.com/NVIDIA/cuda-python/blob/37b401a0b6d8d92dbd4f8b662e02243f9e3e7097/cuda_core/tests/test_event.py#L51-L53

However, that part is made completely redundant by the new tests anyway: better remove?

I think ideally it's best to rename the existing test to `test_timing_success` and to remove the `enable_timing` argument entirely.

Then simply add a couple lines in the new test:

```
    event1 = stream.record(options=enabled)
    event2 = stream.record(options=disabled)
    assert event1.enable_timing is True
    assert event2.enable_timing is False
```